package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Weather-Cast",
+  "version": "1.0.0",
+  "description": "A simple page to check the Weather in any city.",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SigmaSam/Weather-Cast.git"
+  },
+  "keywords": [],
+  "author": "Samuel Almao",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/SigmaSam/Weather-Cast/issues"
+  },
+  "homepage": "https://github.com/SigmaSam/Weather-Cast#readme",
+  "devDependencies": {
+    "babel-eslint": "10.1.x",
+    "eslint": "6.8.x",
+    "eslint-config-airbnb-base": "14.1.x",
+    "eslint-plugin-import": "2.20.x",
+    "stylelint": "13.3.x",
+    "stylelint-config-standard": "20.0.x",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "3.17.x",
+    "webpack": "^5.24.4",
+    "webpack-cli": "^4.5.0"
+  }
```

Comment:
Great work adding web-pack to your project. 👏🏿 

- [ ] Kindly ensure all web pack-related packages are added to the package.json. This will ensure that proper configuration is in place. It seems you missed one. Kindly you should also add `webpack-dev-server` ⬇️ 

![devdep_requirement (1)](https://user-images.githubusercontent.com/32361963/111024490-1903c800-83f0-11eb-8150-14e988eaa5dd.png)